landoapi/wsgi.py

```diff
@@ -6,17 +6,13 @@
 """
 import os
 
-from .app import SUBSYSTEMS, construct_app, construct_treestatus_app, load_config
+from .app import SUBSYSTEMS, construct_app, load_config
 
 # Determine which app to construct by looking for a Treestatus specific env variable.
-app_constructor = (
-    construct_treestatus_app
-    if os.getenv("TREESTATUS_APP") is not None
-    else construct_app
-)
+spec = "treestatus.yml" if os.getenv("TREESTATUS_APP") is not None else "swagger.yml"
```

Comment:
When moving this to django lando, it would be better handled via a command vs an environment variable, but this is an acceptable compromise for now after simplifying the app construction.

However, this config variable should be implemented in `app.load_config` (e.g., in the `config` variable) and not fetched directly from the environment in this module.
```suggestion
spec = config["API_SPEC"]
```

And since that would be done there, it would be better to have it as a string (spec) rather than a boolean.